gapic/api/artman_clouddebugger.yaml

```diff
@@ -0,0 +1,30 @@
+common:
+  api_name: google-clouddebugger-v2
+  proto_gen_pkg_deps:
+    - google-common-protos
+  import_proto_path:
+    - ${REPOROOT}/googleapis
+  src_proto_path:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2
+  service_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/clouddebugger.yaml
+  gapic_api_yaml:
+    - ${REPOROOT}/googleapis/google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml
+  output_dir: ${REPOROOT}/artman/output
+  enable_batch_generation: True
+java:
+  final_repo_dir: ${REPOROOT}/google-cloud-java/google-cloud-clouddebugger
+python:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-python-clouddebugger
+go:
+  final_repo_dir: ${REPOROOT}/gapi-clouddebugger-go
+csharp:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-csharp-clouddebugger
+php:
+  final_repo_dir: ${REPOROOT}/artman/output/gcloud-php-clouddebugger
+ruby:
+  final_repo_dir: ${REPOROOT}/google-cloud-ruby/google-cloud-clouddebugger
+  skip_packman: True
```

Comment:
So in the case of this API, for which the flag isn't set, what happens if, after the generated libraries are published, they want to write a hand-written library on top?

> It seems to me that keeping a machine readable list of APIs that require this flag is doing the exact same thing as setting this flag

Not the same: Setting this flag right now requires 2 lookups and 2 reviews for every API, so multiple hundreds. Listing the APIs with hand-written libraries is writing a list with 11 + 14 entries, that gets updated rarely.
Ask yourself, what would take you less time: list the existing veneers, or check one by one each API to see if it has a veneer? Even if you did the latter, your brain would end up remembering the list of veneers to save you trouble.